sample-git-demo-app/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps.open-cluster-management.io/v1
+kind: Subscription
+metadata:
+  name: demo-subscription
+  namespace: demo-ns-helm-git
+  labels:
```

Comment:
You have to add the annotation that points to the helcm chart within the repo used by the channel

  ```annotations:
    apps.open-cluster-management.io/github-path: resources-roke/helmchart
    apps.open-cluster-management.io/github-branch: master```

Reply:
I talked to Feng and updated the 2 samples that used roke's testrepo to have the annotations